### DISCUSSION TITLE
一种 Obsidian vault 自动备份方法

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#20] [General]
# 一种 Obsidian vault 自动备份方法

介绍一种基于 Git 和 Github Private Repo 自动备份 Obsidian vault 的方法

https://www.edony.ink/auto-backup-obsidian-vault/

**Comment by hinzer** [comment]
移动端 obsidian 如何能通过这种方案配置呢？

**Reply by edonyzpc** [reply]
移动端我的做法是直接在 mac 上配置完成之后先同步到 github，然后再在移动端上通过 [gitdrive](https://apps.apple.com/cn/app/gitdrive-%E5%BC%BA%E5%A4%A7%E7%9A%84git%E5%AE%A2%E6%88%B7%E7%AB%AF%E4%B8%8E%E6%9C%8D%E5%8A%A1%E5%99%A8/id1215974700) 这个 app clone 到指定的移动端本地目录，安卓的话你自己找找有哪些移动端的 git client app 即可
